XRTK.Oculus/Packages/com.xrtk.oculus/Editor/OculusManifestPreprocessor.cs

```diff
@@ -0,0 +1,84 @@
+/************************************************************************************
+
+Copyright   :   Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.
+
+Licensed under the Oculus SDK License Version 3.4.1 (the "License");
+you may not use the Oculus SDK except in compliance with the License,
+which is provided at the time of installation or download, or which
+otherwise accompanies this software in either electronic or hard copy form.
+
+You may obtain a copy of the License at
+
+https://developer.oculus.com/licenses/sdk-3.4.1
+
+Unless required by applicable law or agreed to in writing, the Oculus SDK
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+************************************************************************************/
+
+using UnityEngine;
+using UnityEditor;
+using System.IO;
+
+namespace XRTK.Oculus
+{
+	public class OculusManifestPreprocessor
+	{
+		[MenuItem("Mixed Reality Toolkit/Tools/Create store-compatible AndroidManifest.xml", false, 100000)]
```

Comment:
Not sure I want to add this to the menu or actually put it in the Build Window itself

Reply:
Also this doesn't specify that it's an Oculus specific method in the name of the menu item.

Reply:
Updated